docs/docs/configuration/index.md

```diff
@@ -53,6 +53,9 @@ mqtt:
   client_id: frigate
   # Optional: user
   user: mqtt_user
+  # NOTE: MQTT user can be specified with an environment variables that must begin with 'FRIGATE_'.
```

Comment:
```suggestion
   client_id: frigate
   # Optional: user
  # NOTE: MQTT user can be specified with an environment variables that must begin with 'FRIGATE_'.
   user: mqtt_user
```

Reply:
Comment needs to be above the value.

Reply:
Fixed in latest commit, had not pay attn enough.